addons/website_event_sale/tests/common.py

```diff
@@ -34,28 +39,72 @@ def setUpClass(cls):
         }])
         cls.currency_test = cls.env['res.currency'].create({
             'name': 'eventX',
-            'rate': 10,
             'rounding': 0.01,
             'symbol': 'EX',
         })
         cls.partner = cls.env['res.partner'].create({'name': 'test'})
-        cls.new_company = cls.env['res.company'].create({
-            'currency_id': cls.env.ref('base.EUR').id,
-            'name': 'Great Company EUR',
-            'partner_id': cls.partner.id,
-        })
-        cls.env['res.currency.rate'].create({
-            'company_id': cls.new_company.id,
+
+        cls.env['res.currency.rate'].search([]).unlink()
+        cls.rate = cls.env['res.currency.rate'].create({
+            'company_id': cls.env.company.id,
             'currency_id': cls.currency_test.id,
             'name': '2022-01-01',
-            'rate': 10,
+            'rate': 10.0,
```

Comment:
Is it necessary?

Reply:
No, but while I was it, I tried to respect as mush as how it is done in the code. But I can remove it if you think it create a useless diff.